pandas/core/window.py

```diff
@@ -899,11 +899,45 @@ def skew(self, **kwargs):
         return self._apply('roll_skew', 'skew',
                            check_minp=_require_min_periods(3), **kwargs)
 
-    _shared_docs['kurt'] = """Unbiased %(name)s kurtosis"""
+    _shared_docs['kurt'] = dedent("""Calculate unbiased %(name)s kurtosis.
 
-    def kurt(self, **kwargs):
+This function uses Fisher's definition of kurtosis (kurtosis of normal
+== 0.0) without bias.
```

Comment:
I know this is copied from the DataFrame/Series one, but I personally don't find "kurtosis of normal == 0.0" very clear. I suppose it is about a normal distribution having a kurtosis of 0.0 ?